src/component/misc/validation/InputValidationMessage.scss

```diff
@@ -0,0 +1,10 @@
+$qualityAffectingRuleViolationColor: #8965e0;
```

Comment:
Why not yellow variants are used here?

Reply:
We agreed with @MichalMed that yellow is barely visible. And orange looks too much like error preventing form from being submitted.